docs/src/developing/programming-model/calling-between-programs.md

```diff
@@ -56,11 +56,6 @@ mod acme {
 given instruction to the `token` program via the instruction's `program_id`
 field.
 
-Note that `invoke` requires the caller to pass all the accounts required by the
```

Comment:
I think this is still true, no? Except the executable account doesn't need to be passed in `account_infos` here: https://docs.rs/solana-program/1.8.6/solana_program/program/fn.invoke.html

Reply:
Thanks for the link, the
`The program id of the instruction being issued must also be included in account_infos.`
should be removed as well.

Which part of this note here should stay?

Reply:
The instruction account inputs (excluding the program id account) should still be passed to `account_infos` in the `invoke` call right?

Reply:
They are still translated (at least for BPF program CPI's) with `syscall.translate_accounts`

Reply:
Yes, all `Instruction` accounts except for the first parameter (`program_id`).
I will try to put both notes differently.